lib/srv/termmanager_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package srv
+
+import (
+	"io"
+	"testing"
+	"time"
+	"unsafe"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAtomicAlign(t *testing.T) {
+	m := NewTermManager()
+
+	verifyAlign := func(field *uint64) {
```

Comment:
nice idea!

Reply:
@xacrimon I don't think this test is correct as it works only on 32 bit systems. Look at this example: https://go.dev/play/p/FnTW_KRT__H on 64bit architecture it always passes even if you add something that breaks the alignment.

Reply:
Yep, missed that. Go will forcibly align it to 64-bit regardless on those platforms so this test ends up somewhat worthless.